pkg/apis/pipeline/v1alpha1/task_types.go

```diff
@@ -57,6 +57,11 @@ type TaskSpec struct {
 	// ContainerTemplate can be used as the basis for all step containers within the
 	// Task, so that the steps inherit settings on the base container.
 	ContainerTemplate *corev1.Container `json:"containerTemplate,omitempty"`
+
+	// Host networking requested for this task. Use the host's network namespace.
+	// If this option is set, the ports that will be used must be specified.
```

Comment:
Where must they be specified?

Reply:
I copied this annotation from the `PodSpec` in `k8s.io`. Actually, I just add the `hostNetwork` in my task yaml file and specify nothing. So I think I need remove this annotation. @ImJasonH